extensions/pom.xml

```diff
@@ -117,6 +117,7 @@
 
         <!-- kogito -->
         <module>kogito</module>
+        <module>narayana-stm</module>
```

Comment:
Could you maybe add this one just below Narayana in the `Data access and validation` section?

Reply:
+1